frontend/public/components/monitoring/_monitoring.scss

```diff
@@ -24,6 +28,8 @@
 .monitoring-dashboards__panel {
   height: calc(100% - 20px);
   margin: 0 -5px 20px -5px;
+  max-height: 400px;
```

Comment:
Will this add a scrollbar to some tables? That might be a little awkward since we have paging. I'd think we either want to scroll or page.

Reply:
Thanks @spadgett, good point.

I changed the PR so that `max-height` only applies to panels with `gradient` set to true.